apps/interpreter/src/run-action.ts

```diff
@@ -45,7 +44,7 @@ export async function runAction(
     debug: options.debug,
     debugGranularity: options.debugGranularity,
     debugTarget: options.debugTarget,
-  }).addWorkspace(workingDir);
+  }).addWorkspace('.');
```

Comment:
Maybe consider keeping the current logic and just assigning the string `.` to `workingDir` and commenting there. Is a bit cleaner than a "magic string". Otherwise, godspeed.